src/framework/audio/internal/audiobuffer.cpp

```diff
@@ -108,8 +103,8 @@ void AudioBuffer::setMinSampleLag(unsigned int lag)
         lag = m_data.size();
     }
     m_minSampleLag = lag;
-    if (m_data.size() < 2 * m_minSampleLag * m_streamsPerSample) {
-        m_data.resize(2 * m_minSampleLag * m_streamsPerSample, 0.f);
+    if (m_data.size() < 2 * m_minSampleLag * config()->requiredAudioChannelsCount()) {
+        m_data.resize(2 * m_minSampleLag * config()->requiredAudioChannelsCount(), 0.f);
```

Comment:
I believe, here better remove data resize, else may be data size, not multiple by FILL_SAMLES.
Better add check, maybe even assert, if the size is smaller

Reply:
removed the entire if